test/parallel/test-child-process-stdio-big-write-end.js

```diff
@@ -43,10 +43,10 @@ function parent() {
   child.stdout.on('data', function(c) {
     n += c;
   });
-  child.stdout.on('end', function() {
+  child.stdout.on('end', common.mustCall(function() {
     assert.strictEqual(+n, sent);
```

Comment:
the `+` is cheating the `strict`
either put on the right side `send + '\n'` (`'\n'` is because data comes from `console.log`)
or replace the 'data' handler with:
```js
  let n = 0;
  child.stdout.setEncoding('ascii');
  child.stdout.on('data', function(d) {
    const c = Number(d);
    if (!c) assert.fail(c);
    n += c;
  });
```